src/account-info/AccountDetailsScreen.js

```diff
@@ -29,29 +36,51 @@ class AccountDetailsScreen extends PureComponent<Props> {
   };
 
   render() {
-    const { user } = this.props;
+    const { user, isDeactivated } = this.props;
     const title = {
       text: '{_}',
       values: {
         // This causes the name not to get translated.
         _: user.full_name || ' ',
       },
     };
-
-    return (
-      <Screen title={title}>
-        <AccountDetails user={user} />
+    let deactivatedText = null;
+    let privateMessageButton = null;
+    if (isDeactivated) {
+      deactivatedText = (
+        <RawLabel style={styles.deactivatedText} text="(This user has been deactivated)" />
+      );
+      privateMessageButton = (
```

Comment:
Here both the components are same, just the difference is in `text`. This will look much more cleaner this this
```
text={isDeactivated ? "View private messages" : "Send private message"}
```